src/main/webapp/onelive/archive/index.html

```diff
@@ -115,9 +124,8 @@ <h5 class="font-weight-900 text-dark">
                     </p>
                 </div>
             </div>
-            <a class="btn btn-onelive position-absolute hover-element rounded-05-rem"
-               style="right: 20px; bottom: 20px"
-               href="https://www.youtube.com/watch?v={{snippet.resourceId.videoId}}" target="_blank">
+            <a class="btn btn-onelive position-absolute hover-element rounded-05-rem" href="#youtubeVideoPlayer" onclick="loadYoutubeVideoPlayer(this.name)"
+               style="right: 20px; bottom: 20px;" name="{{snippet.resourceId.videoId}}">
```

Comment:
Hard wrap long lines

Reply:
What is the purpose of adding `name="{{snippet.resourceId.videoId}}"` ?

Reply:
Its purpose is to get the videoId and send it as a parameter to the function